accessing-betydb-with-arcmap.md

```diff
@@ -0,0 +1,157 @@
+# Accessing BETYdb from GIS Software 
+
+Interested researchers can access BETYdb directly from GIS software such as ESRI ArcMap and QGIS. 
+In some cases direct access can simplify the use of spatial data in BETYdb data, but this convenience must be weighed against a more complex setup, limits of GIS software compatibility, and additional complexity of extracting data from a PostGIS SQL database.
+
+## Overview
+
+Accessing the production BETYdb used by the TERRA REF program requires creating a secure shell tunnel (SSH) to a remote server. 
+After creating the tunnel, the database is accessed as if it were available on the local machine. 
+A step-by-step process is given below.
+
+## Configuration used for these instructions
+
+* ArcMap 10.3 or later
+   * ArcMap implies Windows operating system
+* PuTTY: ssh client for Windows that can be downloaded here:  [PuTTY](http://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+
+## Setup
+
+### Request Access 
+
+Request access [to the BETYdb server](https://identity.ncsa.illinois.edu/join/TU49BUUEDM) by following the link. 
+This will take you to the NCSA identity service. If you do not have an NCSA account, you will be asked to create one.
+  This account and password will be used to login to the database server. Access will generally be granted within 24-hours.
+
+### Confirm Access 
+
+Using PuTTY or your preferred SSH client and your NCSA account. 
+
+For PuTTY:
+
+* Launch the PuTTY program
+* click Terminal --> Open
+
+To login to bety6.ncsa.illinois.edu using ssh from the command line:
+
+```sh
+ssh <login>@bety6.ncsa.illinois.edu
+```
+
+After confirming access to bety6 logout by typing `exit`.
+
+### Create SSH Tunnel to BETYdb 
+
+The following command will create an `SSH tunnel` from your computer to the BETYdb server: 
+
+```sh
+ssh -L 5432:localhost:5432 <login>@bety6.ncsa.illinois.edu
+```
```

Comment:
@jterstriep when I try this I get the error

>  ssh -L 5432:localhost:5432 dlebauer@bety6.ncsa.illinois.edu
bind: Operation not permitted
channel_setup_fwd_listener_tcpip: cannot listen to port: 5432
Could not request local forwarding.

Note that I am using MobaXTerm - which is built on PuTTY. I couldn't launch a terminal from the PuTTY software.